### PR TITLE
Bug fixes in custom workflow section and added autocomplete search for experiment selection

### PR DESCRIPTION
**Pull request by amityt**
Signed-off-by: Amit Kumar Das <amitkumar.das@mayadata.io>

## Proposed changes

Minor bug fixes in custom worklfow section and fixed some known issues

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/views/CreateWorkflow/CustomWorkflow/CreateWorkflow/index.tsx [hunk 1]
```diff
@@ -49,7 +55,7 @@ const CreateWorkflow: React.FC<VerifyCommitProps> = ({ gotoStep }) => {
   const classes = useStyles();
   const [allExperiment, setAllExperiment] = useState<ChartName[]>([]);
   const [selectedHub, setSelectedHub] = useState('Public Hub');
-  const [selectedExp, setSelectedExp] = useState('Select');
+  const [selectedExp, setSelectedExp] = useState('Select an experiment');
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/CustomWorkflow/CreateWorkflow/index.tsx:58]: Translation

File: litmus-portal/frontend/src/views/CreateWorkflow/CustomWorkflow/CreateWorkflow/index.tsx [hunk 2]
```diff
@@ -241,51 +257,79 @@ const CreateWorkflow: React.FC<VerifyCommitProps> = ({ gotoStep }) => {
...
+                              }
+                            }}
+                          >
+                            {exp.ExperimentName}
+                          </MenuItem>
+                        ))
+                      ) : (
+                        <MenuItem value="Select an experiment">
+                          No experiments found
+                        </MenuItem>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/CustomWorkflow/CreateWorkflow/index.tsx:328]: Translation

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

**Review by vanshBhatia-A4k9 — APPROVED** [review]
PTAL at suggestions by @ishangupta-ds, I'm unsure if all can be addressed directly :thinking:

**Review by vanshBhatia-A4k9 — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]
/merge

**Review by ishangupta-ds — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Review by vanshBhatia-A4k9 — APPROVED** [review]

**Pull request merged**